core/src/main/java/com/linecorp/armeria/server/RoutingTrie.java

```diff
@@ -77,7 +77,15 @@
      * Returns the list of values which is mapped to the given {@code path}.
      */
     List<V> find(String path) {
-        final Node<V> node = findNode(path, false);
+        return find(path, node -> node);
```

Comment:
How about adding this to `NodeProcessor`?

```java
interface NodeProcessor {
    static <V> NodeProcessor<V> noop() {
        return node -> node;
    }
    ...
}
```

Reply:
Got it. Will add it. Thanks!